pkgs/development/compilers/openjdk/8.nix

```diff
@@ -267,7 +267,7 @@ let
     passthru = {
       inherit architecture;
       home = "${openjdk8}/lib/openjdk";
-      inherit gtk2;
+      gtk = gtk2;
```

Comment:
This change assumes that `gtk2` and `gtk3` are interchangeable, which is not true.

Reply:
Right... Then I'm not committing the last commit. Let's hope people will use these passthrus right..